playbooks/roles/grafana/tasks/main.yml

```diff
@@ -0,0 +1,133 @@
+---
+- name: Include variables
+  include_vars: "{{ lookup('first_found', params) }}"
+  vars:
+    params:
+      files: "{{ distro_lookup_path }}"
+      paths:
+        - "vars"
+
+- name: Install required packages
+  become: true
+  ansible.builtin.package:
+    state: present
+    name: "{{ item }}"
+  loop:
+    - "{{ packages }}"
+  when: "ansible_facts.pkg_mgr != 'atomic_container'"
+  register: task_result
+  until: task_result is success
+  retries: 5
+
+- name: Create Grafana group
+  become: true
+  ansible.builtin.group:
+    name: "{{ grafana_os_user }}"
+    state: present
+
+- name: Create Grafana user for docker
+  become: true
+  ansible.builtin.user:
+    name: "{{ grafana_os_user }}"
+    group: "{{ grafana_os_group }}"
+    groups: docker 
+    state: present
+  when: container_command == "docker"
+
+- name: Create alerta user for podman
+  become: true
+  ansible.builtin.user:
+    name: "{{ grafana_os_user }}"
+    group: "{{ grafana_os_group }}"
+    state: present
+  when: container_command == "podman"
+
+- name: Create required directories
+  become: true
+  file:
+    path: "{{ item.dest }}"
+    state: directory
+    recurse: yes
```

Comment:
please drop this, it is a wrong thing which recursively sets permissions and not `mkdir -p`